src/app/components/TransactionOverview/TransactionOverview.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+import type { Transaction } from '../TransactionForm/TransactionForm'
+
+export type InitialStateType = {
+  transactions: Transaction[]
+}
+
+export default function TransactionOverview({
+  transactions,
+}: InitialStateType): JSX.Element {
+  return (
+    <div>
+      <h2>Überblick:</h2>
+
+      <ul>
+        {transactions.map((transaction: Transaction) => (
+          <li {...(transaction.amount < 0 ? '-' : '')}>
+            <span>
+              {transaction.category}€{Math.abs(transaction.amount)}
```

Comment:
The order of the statements result in a strange output; if I add "Schuhe" for 40€, I get "Schuhe€40" without spaces.
```suggestion
              {transaction.category}: {Math.abs(transaction.amount)}€
```
(or something similar which looks nice)

Reply:
First thank you very much for this review. I still have to do the styling now